packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1266,7 +1356,7 @@ class Auth {
      * A wrapper method for the SLAS endpoint: getPasswordResetToken.
      *
      */
-    async getPasswordResetToken(parameters: ShopperLoginTypes.PasswordActionRequest) {
+    async getPasswordResetToken(parameters: {user_id: string; callback_uri: string}) {
```

Comment:
I'm guessing we changed this to only take in a user_id and callback_uri because everything else is set inside the function?

Reply:
I'm also guessing here since I didn't make the change, but I think the idea is to simplify the API to only ask for the essential user inputs (user_id and callback_uri). 

The function automatically fills in the other required fields like client_id, channel_id (from the configured siteId), mode: 'callback', and hint: 'cross_device' to make it easier for developers to use and prevents configuration errors.

Reply:
@alexvuong makes sense to use the isomorphic types to automatically consume future API changes without code updates. But I've noticed `ShopperLoginTypes.PasswordActionRequest` is no longer available in `commerce-sdk-isomorphic@4.0.0`. Probably that's the reason why Joel was not using it here.
Can you confirm if the equivalent type is `ShopperLoginTypes.getPasswordResetTokenBodyType`? I see that's the one with the same fields. I made the changes to use `ShopperLoginTypes.getPasswordResetTokenBodyType`.